### PR TITLE
Sort incidents on blocked page by priority in descending order

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/155629

### CODE REVIEW

**Comment by okurz** [comment]
decedent? maybe "descending"?

**Comment by okurz** [comment]
Can you provide a screenshot or list for "Show some further details about the incident on the incident page"?

**Comment by Martchus** [comment]
> decedent? maybe "descending"?

I just took the first best spell check suggestion :-)

**Comment by Martchus** [comment]
> Can you provide a screenshot or list for "Show some further details about the incident on the incident page"?

It is nothing fancy. I just wanted to play around with Vue to see how things work and to display the priority value that's actually in the database.

screenshot_20240228_180850

**Review by okurz — APPROVED** [review]

**Review by kraih — DISMISSED** [review]

File: lib/Dashboard/Model/Incidents.pm [hunk 1]
```diff
@@ -256,9 +256,9 @@ sub _update ($self, $db, $incident) {
 
   $db->query(
     'UPDATE incidents SET packages = ?, rr_number = ?, review = ?, review_qam = ?, approved = ?, emu = ?, active = ?,
-       embargoed = ? WHERE id = ?', $incident->{packages}, $incident->{rr_number}, $incident->{inReview},
+       embargoed = ?, priority = ? WHERE id = ?', $incident->{packages}, $incident->{rr_number}, $incident->{inReview},
     $incident->{inReviewQAM}, $incident->{approved}, $incident->{emu}, $incident->{isActive}, $incident->{embargoed},
-    $id
+    $incident->{priority},    $id
```
Comment (b10n1k) [inline lib/Dashboard/Model/Incidents.pm:261]: extra spaces?

Comment (kraih) [reply lib/Dashboard/Model/Incidents.pm:261]: Don't worry about it, we let perltidy decide.

Comment (Martchus) [reply lib/Dashboard/Model/Incidents.pm:261]: Yes. In fact it was tidy's work (possibly to achieve alignment with the list element in the line above).

**Review by okurz — APPROVED** [review]

**Review by perlpunk — APPROVED** [review]

**Pull request merged**